core/src/main/java/com/afollestad/materialdialogs/utils/StringExt.kt

```diff
@@ -9,7 +9,7 @@ import androidx.annotation.ArrayRes
 import androidx.annotation.StringRes
 import com.afollestad.materialdialogs.MaterialDialog
 
-internal fun MaterialDialog.getString(
+fun MaterialDialog.getString(
```

Comment:
This should remain internal, otherwise it gets exposed to consuming apps

Reply:
If you need to access it from another module, it can be redeclared in one of its utility files, even though the duplication isn't ideal

Reply:
Why not change the extension function to a normal utility function instead? Then it could be exposed without polluting the dialog namespace... This seems like something that probably is used in all modules at some time...